tests/utils/postprocessing.py

```diff
@@ -8,8 +8,9 @@
 def get_rounded_result(result: List[Group], round_: int = 2) -> List[tuple]:
     rounded_result = result[0].rows
     for i in range(len(rounded_result)):
-        rounded_result[i] = rounded_result[i][:-1] + \
-                            (round(rounded_result[i][-1], round_),)
+        rounded_result[i] = rounded_result[i][:-2] + \
```

Comment:
I suggest, you give indices names, this probably makes this easier to understand